README.md

```diff
@@ -77,7 +78,9 @@ Given the example above, the code would look like this:
 
 With that set up, we're ready to authenticate a user with GitHub's OAuth API.
 
-With the Sinatra app running, go ahead and visit [localhost:4567](http://localhost:4567) again. This time, clicking the 'Sign In with GitHub' button should take you to an authorization page:
+
+With the Sinatra app running 
+[@ani6gup: I had to restart the server here othwerwise I recieved this URL `https://github.com/login/oauth/authorize?client_id=%3CYOUR_CLIENT_ID`], go ahead and visit [localhost:4567](http://localhost:4567) again. This time, clicking the 'Sign In with GitHub' button should take you to an authorization page:
```

Comment:
Ah yes, it'd be good to mention that the server needs to be restarted for any changes in `server.rb` to take effect.